app/scripts/controllers/authentication/mocks/mockServices.ts

```diff
@@ -0,0 +1,60 @@
+import nock from 'nock';
+import {
+  AUTH_LOGIN_ENDPOINT,
+  AUTH_NONCE_ENDPOINT,
+  LoginResponse,
+  NonceResponse,
+  OAuthTokenResponse,
+  OIDC_TOKENS_ENDPOINT,
+} from '../services';
+
+type MockReply = {
+  status: number;
+  body?: Record<string, unknown>;
+};
```

Comment:
Can we leverage `express.Response` as the type for `mockReply`?

Reply:
This is a simple type used for the `nock` functions below.

I've had a look at express and seems a bit overkill.
1. The Response type is too wide for this simple use case (but we can always `Pick<T>` out the specifics we need.)
2. The Body is too loose being an `any`

It might be fine to keep the type as is or tie it to nock :)